nodejs/week2/src/backend/routes/meals.js

```diff
@@ -0,0 +1,123 @@
+const express = require("express");
+const dataAccess = require("../dataAccess.js");
+const { response } = require("express");
+const router = express.Router();
```

Comment:
as in other files

router is not necessary, you can just export the (req, res) => {} functions and register via app.get in index.js